File: Assembly-CSharp/SRPG/ChatBlackList.cs
// Decompiled with JetBrains decompiler
// Type: SRPG.ChatBlackList
// Assembly: Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null
// MVID: FE644F5D-682F-4D6E-964D-A0DD77A288F7
// Assembly location: C:\Users\André\Desktop\Assembly-CSharp.dll

namespace SRPG
{
  public class ChatBlackList
  {
    public ChatBlackListParam[] lists;
    public int total;

    public void Deserialize(JSON_ChatBlackList json)
    {
      if (json == null)
        return;
      this.lists = (ChatBlackListParam[]) null;
      if (json.blacklist != null)
      {
        this.lists = new ChatBlackListParam[json.blacklist.Length];
        for (int index = 0; index < json.blacklist.Length; ++index)
          this.lists[index] = json.blacklist[index];
      }
      else
        this.lists = new ChatBlackListParam[0];
      this.total = json.total;
    }
  }
}

File: Database_V2/ParamFunctions_untested/ChatBlackList.py
def ChatBlackList(json):
    this={}#ChatBlackListjson)
    #if(json==null)
    #return
    #this.lists=(ChatBlackListParam)null
    #if(json.blacklist!=null)
        #for(intindex=0index<json.blacklist.Length++index)
        if 'blacklist' in json:
            this['lists'] = json['blacklist']
    #else
    #this.lists=newChatBlackListParam[0]
    if 'total' in json:
        this['total'] = json['total']
return this
